composer.json

```diff
@@ -56,7 +56,7 @@
 	"require" : {
 		"clearFw/clearFw" : "~1.0",
 		"easyrdf/easyrdf" : "0.9.*",
-		"doctrine/dbal" : "^2.5",
+		"doctrine/dbal" : "~2.5.0",
```

Comment:
we have to see with @jbout , he introduced that here : https://github.com/oat-sa/generis/pull/316